.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# default group
+* @pcdshub/rix-members
+
+# language-specific group(s)
```

Comment:
There's actually no epics language stuff in this repo. It's not clear to me if epics-reviewers or python-reviewers or neither are appropriate for this repo.

Reply:
I'll put epics-reviewers in the default group as well, and that should be sufficient